automation/deploy-bin.ts

```diff
@@ -59,17 +62,12 @@ export async function release() {
 	}
 }
 
-let cachedOctokit: any;
-
 /** Return a cached Octokit instance, creating a new one as needed. */
-function getOctokit(): any {
-	if (cachedOctokit) {
-		return cachedOctokit;
-	}
-	const Octokit = require('@octokit/rest').plugin(
+const getOctokit = _.once(function() {
```

Comment:
This change isn't strictly necessary, but it makes the code simpler and avoided me having to figure out the correct type for `cachedOctokit` (it's now inferred automatically via `require('@octokit/rest') as typeof _ocktokit`)